packages/beacon-node/src/network/network.ts

```diff
@@ -408,7 +408,7 @@ export class Network implements INetwork {
     };
     const sentPeers = await this.core.publishGossip(topicStr, messageData, opts);
 
-    this.logger.verbose("Publish to topic", {topic: topicStr, sentPeers});
+    this.logger.verbose("Publish to topic", {topic: topicStr, sentPeers, slot: this.clock.currentSlot});
```

Comment:
Not every message includes a message slot, e.g. voluntary  exit or bls to execution change, that's why I opted to use the clock slot instead when the message was published.

To clarify this, we could also do `clockSlot: this.clock.currentSlot` or `currentSlot: this.clock.currentSlot`, but curious what others think

Reply:
Personally think the clarification of `clockSlot` or other might be helpful here.

Reply:
`currentSlot` seems to be the more commonly used, and I think we only use it for the current slot in terms of time-based, whereas `slot` could be message slot or time-based slot